README.md

```diff
@@ -1 +1,19 @@
+# C# - Code Snippets
+This project serves as a guide to help you build an application with FreeClimb. Each snippet represents a different example of a FreeClimb tutorial.
 
+## Setting up your new app within your FreeClimb account
+
+To get started using a FreeClimb account, follow the instructions [here](https://docs.freeclimb.com/docs/getting-started-with-freeclimb).
+
+## Setting up the FreeCLimb SDK
+
```

Comment:
Actually the line about installing the sdk should be left in (especially since you changed the title of this section to setting up the freeclimb sdk)

Reply:
Readded those lines